src/ucp/core/ucp_ep.c

```diff
@@ -3415,6 +3416,73 @@ void ucp_ep_reqs_purge(ucp_ep_h ucp_ep, ucs_status_t status)
     }
 }
 
+static ucs_status_t ucp_ep_query_transport(ucp_ep_h ep, ucp_ep_attr_t *attr)
+{
+    uct_tl_resource_desc_t *rsc;
+    ucp_transport_entry_t *transport_entry;
+    size_t device_limit;
+    size_t transport_limit;
+    ucp_lane_index_t i;
+    ucp_rsc_index_t cm_idx;
+    ucp_rsc_index_t rsc_idx;
+    unsigned max_entries;
+
+
+    ucp_worker_h worker     = ep->worker;
+    ucp_ep_config_t *config = ucp_ep_config(ep);
```

Comment:
init those vars in declaration

Reply:
I'm not sure what you were asking. I think you wanted me to move these lines into declarations at the top of the function.